providers/db.rb

```diff
@@ -39,7 +39,7 @@
 	        :pma_db => new_resource.pma_database,
 	        :auth_type => new_resource.auth_type
 		})
-		mode 00640
+		mode 00644
```

Comment:
The reason the permissions were set in that way is that php-fpm does not need "other" permissions to read that file but in your case, using Apache, it does (apache does not run with the file owner's permission but with www-data/httpd). You should add a check for node['phpmyadmin']['fpm'] there to set the appropriate permissions.